Core/Object Arts/Dolphin/System/Recent/UI.AbstractRecentMenu.cls

```diff
@@ -45,7 +45,7 @@ notEmpty
 	^self getRecentList notEmpty!
 
 onAboutToDisplayMenu: recentMenu
-	"Private - The system is about to display the <Menu>, popup, this is our chance
+	"Private - The system is about to display the <Menu> popup, this is our chance
```

Comment:
The system is about to display the `Menu`, recentMenu, ...

Reply:
Ah, that clarifies that sentence, sort of. I would perhaps have put:  the <Menu>, <recentMenu> popup,... but that's nearly as confusing as not quoting it at all. I might have even taken a leaf out of other programming and put <Menu>->recentMenu but that really doesn't parse well either.